src/target/source/source_module.cc

```diff
@@ -26,6 +26,9 @@
 #include <tvm/runtime/ndarray.h>
 #include <tvm/runtime/packed_func.h>
 #include <tvm/runtime/registry.h>
+#include <tvm/tir/function.h>
+#include <tvm/tir/stmt_functor.h>
+#include <tvm/tir/transform.h>
```

Comment:
It's likely that these are the reason why your Windows CI builds are failing as iirc `source_module.cc` only gets linked with the runtime, not the entirety of TVM?

Reply:
Thanks! lets see. I was scratching my head over this.